app/styles/noscript.css

```diff
@@ -0,0 +1,23 @@
+.no-js-container {
```

Comment:
All other classes in this app are prefixed with "yc-", so these classes should start with "yc-no-js-", instead.